.github/workflows/continuous-integration.yml

```diff
@@ -300,10 +300,11 @@ jobs:
       run: poetry build
 
     - name: Publish to PyPI 📦
-      uses: pypa/gh-action-pypi-publish@37e305e7413032d8422456179fee28fac7d25187
+      uses: pypa/gh-action-pypi-publish@bea5cda687c2b79989126d589ef4411bedce0195
```

Comment:
Is there a reason we use the commit hashes rather than releases?

Reply:
Yes, we use hashes for GH action for security reasons, we wanna be sure that someone didn't re-tag, or did something odd.